File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        index_one=0
        index_two=len(numbers)-1

        while index_one<index_two:
            tmp=numbers[index_one]+numbers[index_two]
            if tmp==target:
                return [index_one+1,index_two+1]
            elif tmp<target:
                index_one+=1
                continue
            elif tmp>target:
                index_two-=1

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int indexone=0;
        int indextwo=numbers.length-1;
        while (indexone<indextwo){
            int value=numbers[indexone]+numbers[indextwo];
            if (value<target){
                indexone++;
            }
            else if(value>target){
                indextwo--;
            }
            else{
                break;
            }   
        }
        int[] res= {indexone+1,indextwo+1};
        return res;
    }
}